### PR TITLE
chore: update project slogan

### PR DESCRIPTION
**Pull request by tisonkun**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

Rendered version https://github.com/tisonkun/greptimedb/tree/slogan.

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

## Summary by CodeRabbit

- **Documentation**
  - Updated the README to reflect rebranding and refocusing of the time-series database application.
  - Emphasized the new unified approach for handling metrics, events, and logs.
  - Highlighted new core features: cloud-edge collaboration and easy horizontal scaling.

### CODE REVIEW

**Comment by tisonkun** [comment]
@sunng87 @killme2008 Please review this slogan and description updates.

I'll update the portal page of doc site and the GitHub org profile once this is accepted.

File: README.md [hunk 1]
```diff
@@ -50,24 +50,23 @@
 
 ## Introduction
 
-**GreptimeDB** is an open-source time-series database focusing on efficiency, scalability, and analytical capabilities.
-Designed to work on infrastructure of the cloud era, GreptimeDB benefits users with its elasticity and commodity storage, offering a fast and cost-effective **alternative to InfluxDB** and a **long-term storage for Prometheus**.
+**GreptimeDB** is an open-source unified time-series database for **metrics**, **events**, and **logs**. You can gain real-time insights from Edge to Cloud at any scale.
```
Comment (killme2008) [inline README.md:53]: Adds `trace in plan`?

File: README.md [hunk 2]
```diff
@@ -50,24 +50,23 @@
...
+* **Unified all kinds of time series**
 
-  Seamless scalability from a standalone binary at edge to a robust, highly available distributed cluster in cloud, with a transparent experience for both developers and administrators.
+  GreptimeDB treats all time series as contextual events with timestamp, and thus unifies the processing of metrics and events. It supports analyzing metrics and events with SQL and PromQL, and doing streaming with continuous aggregation.
 
-* **Analyzing time-series data**
+* **Cloud-Edge collaboration**
 
-  Query your time-series data with SQL and PromQL. Use Python scripts to facilitate complex analytical tasks.
+  GreptimeDB can be deployed on ARM architecture-compatible Android/Linux systems as well as cloud environments from various vendors. Both sides run the same software, providing identical APIs and control plane, so your application can run at the edge or on the cloud without modification.
```
Comment (killme2008) [inline README.md:65]:
```suggestion
  GreptimeDB can be deployed on ARM architecture-compatible Android/Linux systems as well as cloud environments from various vendors. Both sides run the same software, providing identical APIs and control planes, so your application can run at the edge or on the cloud without modification, and data synchronization also becomes extremely easy and efficient.
```

File: README.md [hunk 3]
```diff
@@ -50,24 +50,23 @@
...
 
-* **Cloud-native distributed database**
+* **Easy horizontal scaling**
```
Comment (killme2008) [inline README.md:67]: Keep the title not changing. I think `cloud-native` is more proper here.

File: README.md (hunk 1 above)
Comment (tisonkun) [reply README.md:53]:
```suggestion
**GreptimeDB** is an open-source unified time-series database for **metrics**, **events**, and **logs** (also **trace** in plan). You can gain real-time insights from Edge to Cloud at any scale.
```

File: README.md (hunk 3 above)
Comment (tisonkun) [reply README.md:67]:
```suggestion
* **Cloud-native distributed database**
```

**Comment by tisonkun** [comment]
@killme2008 Updated.

File: README.md [hunk 4]
```diff
@@ -50,24 +50,23 @@
 
 ## Introduction
 
-**GreptimeDB** is an open-source time-series database focusing on efficiency, scalability, and analytical capabilities.
-Designed to work on infrastructure of the cloud era, GreptimeDB benefits users with its elasticity and commodity storage, offering a fast and cost-effective **alternative to InfluxDB** and a **long-term storage for Prometheus**.
+**GreptimeDB** is an open-source unified time-series database for **metrics**, **events**, and **logs** (also **trace** in plan). You can gain real-time insights from Edge to Cloud at any scale.
```
Comment (tisonkun) [inline README.md:53]:
```suggestion
**GreptimeDB** is an open-source unified time-series database for **Metrics**, **Events**, and **Logs** (also **Traces** in plan). You can gain real-time insights from Edge to Cloud at any scale.
```

**Review by sunng87 — APPROVED** [review]

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**